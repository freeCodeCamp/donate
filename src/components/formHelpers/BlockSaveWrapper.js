import React from 'react';
import PropTypes from 'prop-types';

const propTypes = {
  children: PropTypes.node
};

const style = {
  padding: '0 15px'
};

function BlockSaveWrapper({ children }) {
  return <div style={style}>{children}</div>;
}

BlockSaveWrapper.displayName = 'BlockSaveWrapper';
BlockSaveWrapper.propTypes = propTypes;

export default BlockSaveWrapper;
