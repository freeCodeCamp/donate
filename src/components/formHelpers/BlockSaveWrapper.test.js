/* global expect */

import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import BlockSaveWrapper from './BlockSaveWrapper';

Enzyme.configure({ adapter: new Adapter() });

test('<BlockSaveWrapper /> snapshot', () => {
  const component = renderer.create(<BlockSaveWrapper />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
