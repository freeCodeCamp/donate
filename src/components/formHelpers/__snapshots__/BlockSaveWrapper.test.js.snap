// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockSaveWrapper /> snapshot 1`] = `
<div
  style={
    Object {
      "padding": "0 15px",
    }
  }
/>
`;
